src/hostapi/jack/pa_jack.c

```diff
@@ -451,6 +451,49 @@ BlockingWaitEmpty( PaStream *s )
 
 /* ---- jack driver ---- */
 
+void replace_string(char *string, char *to_find, char *replacement)
+{
+    char *found_position = strstr(string, to_find);
+    while( found_position ) {
+        size_t sizeof_original = strlen(string);
```

Comment:
The name of the variable is misleading, because sizeof normally includes the null terminator, len not.
```suggestion
        size_t original_len = strlen(string);
```